warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoFunctionsDescriptor.java

```diff
@@ -128,7 +144,7 @@ public JexlNode getIndexQuery(ShardQueryConfiguration config, MetadataHelper hel
                                         BoundedRange.create(JexlNodeFactory.createAndNode(Arrays.asList(geLatNode2, leLatNode2)))));
                         
                         // link em up
-                        returnNode = JexlNodeFactory.createAndNode(Arrays.asList(andNode1, andNode2));
+                        returnNode = JexlNodeFactory.createOrNode(Arrays.asList(andNode1, andNode2));
```

Comment:
oh interesting.... I guess we would have been missing hits here.  However this situation is probably extremely rare.

Reply:
Also can't this section reuse the code above (if args size == 3)?  As far as I can tell you are simply parsing the coordinates from the arguments slightly differently.  Isn't the subsequent jexl tree essentially the same except that you are not  "optimizing"?  Speaking of which, why aren't we optimizing the ranges in this case?

Reply:
Yeah, I was surprised by the use of and over or. Not sure how we missed that all these years. This case is different. This path doesn't use the Geo index at all. This sets up individual bounded range queries against the individual lat and long fields, so the optimizations don't apply at all.